libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1463,6 +1471,10 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 		break;
 	case Token::Mul:
 		m_context << Instruction::MUL;
+		if (fixedType)
+			m_context << u256(pow(bigint(10), fixedType->fractionalDigits()))
```

Comment:
Unfortunately, it's not that simple. This works for fixed point types with up to 128 bits, but you have to deal with overflow if you have larger numbers.

Reply:
I would propose to remove this feature for now, throw a "not implemented" exception for all operations where overflow might happen for fixed point operands and deal with it in a subsequent pull request.

Reply:
Another thing we could do here is to change the result type of the operation, but we don't do that for integers, so it would probably be weird.

Reply:
I think it's slightly more complicated than 128 bits. For example, a 248 bit fixed point with only 1 decimal digit could safely be shifted. I think the formula is `canScale = bits + ceil(log2(pow(10, fractionalDigits))) <= 256`

Reply:
This also affects exponentiation and division